app/role_data.go

```diff
@@ -175,6 +175,10 @@ func addRoles(management *config.ManagementContext) (string, error) {
 		addRule().apiGroups("management.cattle.io").resources("catalogtemplateversions").verbs("*").
 		addRule().apiGroups("monitoring.cattle.io").resources("prometheus").verbs("view").
 		addRule().apiGroups("monitoring.coreos.com").resources("prometheuses", "prometheusRules", "serviceMonitors").verbs("*").
+		addRule().apiGroups("networking.istio.io").resources("destinationrules", "envoyfilters", "gateways", "serviceentries", "sidecars", "virtualservices").verbs("*").
+		addRule().apiGroups("config.istio.io").resources("apikeys", "authorizations", "checknothings", "circonuses", "deniers", "fluentds", "handlers", "kubernetesenvs", "kuberneteses", "listcheckers", "listentries", "logentries", "memquotas", "metrics", "opas", "prometheuses", "quotas", "quotaspecbindings", "quotaspecs", "rbacs", "reportnothings", "rules", "solarwindses", "stackdrivers", "statsds", "stdios").verbs("*").
+		addRule().apiGroups("authentication.istio.io").resources("policies", "meshpolicies").verbs("*").
+		addRule().apiGroups("rbac.istio.io").resources("clusterrbacconfigs", "rbacconfigs", "serviceroles", "servicerolebindings").verbs("*").
```

Comment:
@loganhz If we want to grant all istio resources, there are also `mesh/v1alpha1`, `policy/v1beta1` 
Can you check if we can use wildcards so that we don't miss anything, e.g. `apiGroups("*.istio.io").resources("*")`

Reply:
I would like to give the permission which kiali is using. So I don't want to use *